src/helpers/buildWalletSections.tsx

```diff
@@ -395,13 +395,48 @@ const withBriefBalanceSection = (
 
   return [
     {
-      type: 'ASSETS_HEADER',
-      uid: 'assets-header',
+      type: 'PROFILE_STICKY_HEADER',
+      uid: 'assets-profile-header-compact',
       value: totalValue,
     },
     {
-      type: 'ASSETS_HEADER_SPACE_AFTER',
-      uid: 'assets-header-space-after',
+      type: 'PROFILE_AVATAR_ROW_SPACE_BEFORE',
```

Comment:
why won't you just add spacing inside the components?

Reply:
I know we have used this somewhere, but I still don't get this. 

This means additional overhead on our fragile recycling logic and why this is better than just adding margins or padding?

Reply:
Yeah, I just did this because it was already a pattern being used (e.g. `NFT_SPACE_AFTER`, `NFT_HEADER_SPACE_AFTER`, etc), plus our design system philosophy is to avoid spacing in child components, but we can make an exception here if it has an effect on our recycling logic. I can change this!